Resources/doc/serialization-groups-and-relations.md

```diff
@@ -215,5 +215,22 @@ services:
         class: Symfony\Component\Serializer\NameConverter\CamelCaseToSnakeCaseNameConverter
 ```
 
+### Entity identifier case
```

Comment:
You mean @ORM\Id ? 

And the custom loader is actually DoctrineIdentifierLoader but I have no other example ...